data_steward/validation/main.py

```diff
@@ -116,18 +115,59 @@ def run_export(hpo_id=None, folder_prefix="", target_bucket=None):
 
     # Run export queries and store json payloads in specified folder in the target bucket
     reports_prefix = folder_prefix + ACHILLES_EXPORT_PREFIX_STRING + datasource_name + '/'
-    for export_name in common.ALL_REPORTS:
-        sql_path = os.path.join(export.EXPORT_PATH, export_name)
+    datasources_json_reports_prefix = folder_prefix + ACHILLES_EXPORT_DATASOURCES_JSON
+    results = run_achilles_export(hpo_id, target_bucket, reports_prefix,
+                                  datasources_json_reports_prefix)
+    return results
+
+
+def run_export_dashboard_bucket(hpo_id, target_bucket=None):
+    """
+    Uploads achilles files to a bucket for dashboards access
+
+    :param hpo_id: which hpo bucket do these files go into
+    :param target_bucket: Name of the achilles metrics transfer board
+    :return:
+    """
+    datasource_name = hpo_id
+    if target_bucket is None:
+        target_bucket = gcs_utils.get_achilles_transfer_bucket()
+
+    logging.info(
+        f'Exporting {datasource_name} report to bucket {target_bucket}')
+
+    # Run export queries and store json payloads in specified folder in the target bucket
+    folder_prefix = hpo_id + '/'
```

Comment:
Might be a good idea to use `os.path.join` instead of concatenating the string ourselves